terraform/modules/upload-service/health_check_lambda.tf

```diff
@@ -92,7 +92,7 @@ resource "aws_lambda_function" "upload_health_check_lambda" {
 }
 
 resource "aws_cloudwatch_event_rule" "daily" {
-    name = "every-day-${var.deployment_stage}"
+    name = "upload-every-day-${var.deployment_stage}"
```

Comment:
Since we're already in here can we namespace it as dcp-upload-daily-health-check-${var.deployment_stage} to match other names and be more explicit?

Reply:
sure